win_configure/msi/wix/prep.bat

```diff
@@ -91,6 +91,15 @@ if %ERRORLEVEL% == 0 (
 	xcopy /s /d "%%~a" "%EXECDIR%\bin" > NUL
 	if NOT %ERRORLEVEL% == 0 goto theend
 	)
+	REM Exclude list of files from bin directory
+	For %%b in (
+		"%EXECDIR%\bin\pbs_dataservice.bat"
+		"%EXECDIR%\bin\pbs_ds_monitor.exe"
+		"%EXECDIR%\bin\pbs_ds_password.exe"
+	) do (
+	del /s "%%~b" > NUL
+	if NOT %ERRORLEVEL% == 0 goto theend
```

Comment:
Indentation looks off here.

Reply:
Thanks for your comment @mike0042 , I have fixed indentation issue. 
Please have a look and share your further comments/sign-off